test/parallel/test-tls-transport-destroy-after-own-gc.js

```diff
@@ -0,0 +1,32 @@
+// Flags: --expose-gc
+'use strict';
+
+// Regression test for https://github.com/nodejs/node/issues/17475
+// Unfortunately, this tests only "works" reliably when checked with valgrind or
+// a similar tool.
+
+/* eslint-disable no-unused-vars */
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
```

Comment:
This is actually a unused var. As far as I see it you only wanted to deactivate it for `clientTLSHandle`. So you could just use `// eslint-disable-next-line no-unused-vars`?

Reply:
done!